### PR TITLE
Deploy devo (gamma) serverless demo that talks to gamma Chime endpoint

### PR DESCRIPTION
**Pull request by tianyux-amz**
**Issue #:** N/A

**Description of changes:** Add an optional parameter to the serverless demo deployment script to specify Chime endpoint, and deploy to a new devo canary that talks to gamma Chime endpoint

**Testing**

1. Have you successfully run `npm run build:release` locally?
   - Yes
2. How did you test these changes?
   - Was able to deploy serverless demo and meeting readiness checker that talk to gamma endpoints. Verified in choc that the demos were talking to media in gamma stage. Ran sanity test on the gamma serverless demo.
3. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
   - No
4. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
   - No

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: script/deploy-canary-demo [hunk 1]
```diff
@@ -12,10 +12,12 @@ echo "Deploying to alpha stage for canary"
 npm run deploy -- -b chime-sdk-demo-canary -s chime-sdk-demo-canary
 npm run deploy -- -b chime-sdk-meeting-readiness-checker-dev-canary -s chime-sdk-meeting-readiness-checker-dev-canary -a meetingReadinessChecker
 
-
-echo "Deploying to gamma stage for canary"
-
+echo "Deploying to devo stage (that talks to gamma Chime endpoint) for canary"
 export AWS_ACCESS_KEY_ID=$GAMMA_AWS_ACCESS_KEY_ID
```
Comment (ltrung) [inline script/deploy-canary-demo:16]: This is not correct right? Both of this deploy to the same aws account (chime-sdk-gamma-canary) and not the devo stage.

Comment (asnyatkov) [reply script/deploy-canary-demo:16]: +1 Remove stage mentions

Comment (tianyux-amz) [reply script/deploy-canary-demo:15]: Sorry if the wording was not accurate. I meant the to-be-added devo stage in our canary pipeline. Rephrased a little bit.

Comment (tianyux-amz) [reply script/deploy-canary-demo:15]: Discussed offline. We will create a new AWS account for the devo (gamma-endpoint) canary. Will update the PR accordingly.

**Review by ltrung — APPROVED** [review]

**Pull request merged**